Tests/Marketplace/marketplace_services.py

```diff
@@ -629,6 +629,33 @@ def _load_pack_dependencies(self, index_folder_path, first_level_dependencies, a
 
         return dependencies_data_result
 
+    def _get_updated_changelog_entry(self, changelog: dict, version: str, release_notes: str = None,
+                                     version_display_name: str = None, build_number_with_prefix: str = None,
+                                     released_time: str = None):
+        """
+
+        Args:
+            changelog (dict): The changelog from the production bucket.
+            version (str): The version that is the key in the changelog of the entry wished to be updated.
+            release_notes (str): The release notes lines to update the entry with.
+            version_display_name (str): The version display name to update the entry with.
+            build_number_with_prefix(srt): the build number to modify the entry to, including the prefix R (if present).
+            released_time: The released time to update the entry with.
+
+        """
+
+        changelog_entry = changelog[version]
+        version_display_name = \
+            version_display_name if version_display_name else changelog_entry['displayName'].split('-')[0]
+        build_number_with_prefix = \
+            build_number_with_prefix if build_number_with_prefix else changelog_entry['displayName'].split('-')[1]
+
+        changelog_entry['releaseNotes'] = release_notes if release_notes else changelog_entry['releaseNotes']
+        changelog_entry['displayName'] = f'{version_display_name} - {build_number_with_prefix}'
+        changelog_entry['released'] = released_time if released_time else changelog_entry['released']
```

Comment:
When developing function in mp_services we try to use global constants that are maintained in `marketplace_constants.py`. Can you please add a new class named `Changelog` and add all these constants? if so please replace the existing ones in `create_changelog_entry` func keep on consistency.